packages/web-worker/README.md

```diff
@@ -14,25 +14,25 @@ $ yarn add @shopify/web-worker
 
 ## Usage
 
-This library contains three parts that must be used in tandem:
+This library contains three parts that must be used together:
 
 1. The public API of the package provided by `@shopify/web-worker`
-1. A webpack plugin provided by `@shopify/web-worker/webpack`
-1. A babel plugin provided by `@shopify/web-worker/babel`
+1. A babel plugin provided by `@shopify/remote-ui/babel` that identifies uses of `createWorkerFactory` that need to be processed
+1. A webpack plugin provided by `@shopify/remote-ui/webpack` that outputs the worker as a dedicated chunk
```

Comment:
On first glance of this diff it sounded like `@shopify/web-worker/{babel,webpack}` were removed but they're still present and referenced several times in this readme. Should this have been kept the same?